packages/0x.js/src/types.ts

```diff
@@ -332,6 +332,7 @@ export interface TxOpts {
     from: string;
     gas?: number;
     value?: BigNumber;
+    gasPrice?: BigNumber;
 }
```

Comment:
This type needs a comment above it, explaining each param. I know it seems obvious to us now, but let's think of the beginners ;)

Reply:
It's a private type